.github/workflows/openapi.yaml

```diff
@@ -5,14 +5,24 @@ on:
     branches:
       - main
     paths:
+      # TODO: remove old ones
       - auth-server-open-api-spec.yaml
       - open-api-spec.yaml
```

Comment:
Any reason not to go ahead and remove the old files?

Reply:
Not really. I kept them for reference. Deleted them now.